readthedocs.yaml

```diff
@@ -0,0 +1,20 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools: {python: "3.8"}
+  jobs:
+    post_install:
+      - pip install poetry>=1.2.1
+      - poetry config virtualenvs.create false
+      - poetry install --with docs
```

Comment:
OK this is going to depend on a poetry group called `docs` which isn't present in `pyproject.toml` . You can also read here for more info: https://python-poetry.org/docs/managing-dependencies/#dependency-groups and probably dask-ms's `pyproject.toml` for how to create it.

Groups don't have a `setup.py` correspondence, they're a new poetry feature. There's potential for confusion here.

Note that poetry also has the concept of [extras](https://python-poetry.org/docs/pyproject#extras) which *does* correspond to `setup.py`'s `extras_require` feature